src/Template/Layout/js/app/app.js

```diff
@@ -126,6 +126,24 @@ const _vueInstance = new Vue({
     methods: {
         // Events Listeners
 
+        /**
+         * Clone object
+         * Prompt for title change
+         *
+         * @param {Event} e The event
+         * @return {void}
+         */
+        clone(e) {
+            const title = document.getElementById('title').value;
+            const cloneTitle = prompt(title, title + ' -copy');
```

Comment:
User prompt should tell something more like:
`Please insert a new title on "{title}" clone"